regulations/views/preamble.py

```diff
@@ -47,14 +47,15 @@ def get(self, request, *args, **kwargs):
             raise Http404
 
         context = generate_html_tree(subtree)
-        template = context['node']['template_name']
         context['use_comments'] = True
+        template = context['node']['template_name']
 
+        context = {'sub_context': context, 'sub_template': template,
+                   'preamble': preamble}
         if not request.is_ajax() and request.GET.get('partial') != 'true':
-            # Wrap the inner context
-            context = {'sub_context': context, 'sub_template': template,
-                       'preamble': preamble}
             template = 'regulations/preamble-chrome.html'
+        else:
+            template = 'regulations/preamble-partial.html'
```

Comment:
You're right, that is how the regs stuff works -- I'd argue it's wonky, though, and I was hoping the system here would be able to replace it. I think we architected a lot of funkiness originally, including the partial templates. No biggie regarding continuing the regs style; just trying to clean stuff up.

If you can think of a better way to integrate front and backend (maybe the `partial` endpoint can include both sidebar and footer markup, split out when it's received by the frontend, for example?), let's work on implementing. Otherwise, we can make do.